start-singleuser.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I would prefer not to duplicate the `start-singleuser.sh`, can we rather create an executable `run` script here https://github.com/opendatahub-io/s2i-lab-elyra/tree/master/.s2i/bin which would run the setup and then `exec /usr/libexec/s2i/run` ?